src/components/organisms/SearchModal.tsx

```diff
@@ -82,7 +127,7 @@ const SearchModal = ({ storage }: SearchModalProps) => {
 
   const handleSearchInputKeyDown = useCallback(
     (event: KeyboardEvent<HTMLInputElement>) => {
-      console.log(event.key)
+      // console.log(event.key)
```

Comment:
Please discard it.

Reply:
Discarded, is there any reason (testing?) as why we want this?

Reply:
I don't like to keep any unnecessary code. Once we finish our work, we should tidy up!

Reply:
So does discard it means remove the line, or don't modify it? Because the console log was there, it printout each character user presses for search in console. That's why I asked do we want this printout, or I should remove it? :D

Reply:
I'll discard it!